pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- Example: list unneeded packages with deborphan and remove them with apt-get:
+- delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- assign incoming arguments to a variable so that they may be used in the middle of a command
```

Comment:
I actually disagree with this explanation. It does not tell what the whole command does (copying files starting with M to Desktop).

If example is too complicated for explanation, let's avoid adding it all. Of course, if you can provide good explanation, then do it.